webdriver-spec.html

```diff
@@ -4319,7 +4319,7 @@ <h2 data-dfn-for="NavigatorAutomationInformation">Interface</h2>
 
 <p>A <dfn data-lt="keyboard-interactable">keyboard-interactable element</dfn>
  is any <a>element</a> that has a <a>focusable area</a>,
- or is the <a><code>body</code> element</a>,
+ is a <a><code>body</code> element</a>,
```

Comment:
I don't think that fixes the issue. It appears to make body or document elements keyboard interactable. I think we just need to say that `<a><code>body</code> element</a> if the document's namespace is html` linking to https://infra.spec.whatwg.org/#html-namespace